src/translations/index.js

```diff
@@ -22,28 +31,36 @@ var fiveCharacterLocales = {
   pt_BR: require('./pt_BR'),
   pt_PT: require('./pt_PT'),
   ru_RU: require('./ru_RU'),
+  sk_SK: require('./sk_SK'),
   sv_SE: require('./sv_SE'),
   th_TH: require('./th_TH'),
   zh_CN: require('./zh_CN'),
   zh_HK: require('./zh_HK'),
-  zh_TW: require('./zh_TW')
+  zh_TW: require('./zh_TW'),
+  zh_XC: require('./zh_XC')
 };
 
 var twoCharacterLocaleAliases = {
+  ar: fiveCharacterLocales.ar_EG,
+  cs: fiveCharacterLocales.cs_CZ,
   da: fiveCharacterLocales.da_DK,
   de: fiveCharacterLocales.de_DE,
+  el: fiveCharacterLocales.el_GR,
   en: fiveCharacterLocales.en_US,
   es: fiveCharacterLocales.es_ES,
+  fi: fiveCharacterLocales.fi_FI,
   fr: fiveCharacterLocales.fr_FR,
```

Comment:
Curious - how do you choose which localization gets the 2 letter alias. For example for `fr`, how did you decide to use `fr_FR` instead of `fr_XC`. Are these standards listed somewhere for reference?

Reply:
We prefer the base language over the variant language when a merchant selects the locale using only 2 characters. So if `fr` is passed as the locale, we assume they want `fr_FR` rather than the variant.

Reply:
Thanks!